### DISCUSSION TITLE
enable transition for extenstion?

### DISCUSSION THREAD

**Discussion by txtl** [visgl/deck.gl#8196] [General]
I implemented an extension to render dash color for PathLayer, and I've not been able to get transition to work even though I use transition: true, in addInstanced:

attributeManager.addInstanced({
        instanceDashColors: {size: this.props.colorFormat.length, transition: true, accessor: 'getDashColor'}
      });

Is there anything else needed to enable transition? 

Thanks,

**Comment by Pessimistress** [comment]
Please create a Codepen/Code Sandbox.
